apidoc/Titanium/App/Android/Android.yml

```diff
@@ -35,6 +35,15 @@ properties:
     permission: read-only
     since: 3.3.0
 
+events:
+  - name: shortcutitemclick
+    summary: Fired when a <Titanium.UI.ShortcutItem> is clicked.
+    properties:
+      - name: id
+        summary: Identifier of the clicked shortcut item.
+        type: String
+    since: 7.1.0
```

Comment:
This looks outdated. The `shortcutitemclick` event should be fired on the `Ti.App` instance.

Reply:
Needs to be 7.3.0 now.